mico-core/src/main/java/io/github/ust/mico/core/model/MicoApplication.java

```diff
@@ -125,46 +89,30 @@
      * The list of service deployment information
      * this application uses for the deployment of the required services.
      */
-    @ApiModelProperty(extensions = {@Extension(
-        name = CustomOpenApiExtentionsPlugin.X_MICO_CUSTOM_EXTENSION,
-        properties = {
-            @ExtensionProperty(name = "title", value = "Service Deployment Information"),
-            @ExtensionProperty(name = "x-order", value = "130"),
-            @ExtensionProperty(name = "description", value = "The list of service deployment information " +
-                "this application uses for the deployment of the required services.")
-        }
-    )})
-    @JsonManagedReference
```

Comment:
I think `@JsonManagedReference` is still relevant here

Reply:
I'm not 100% sure, i will just add it again.

Reply:
It's necessary if the `MicoApplication` is serialized with Jackson. So it would be better to have it.

Reply:
Yes, with the new DTOs  I think we do not serialize `MicoApplication`instances, but let's keep it in case ...